content/300-accelerate/350-static-ip.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Static IP"
+metaTitle: "Enable Static IP for Prisma Accelerate"
+metaDescription: "Learn enabling Static IP for Prisma Accelerate."
+---
+
+You can enable static IP for Accelerate when your security setup requires IP allowlisting or if you're implementing firewalls that only permit access from trusted IPs, ensuring controlled and secure database connections.
+
+![](./images/result-of-adding-static-ip-to-accelerate.png)
```

Comment:
let's make sure to add alt text whenever possible.

Reply:
+1